pootle/core/search/base.py

```diff
@@ -16,6 +16,8 @@
 class SearchBackend(object):
     def __init__(self, config_name=None, **kwargs):
         self._setup_settings(config_name)
+        self.weight = 1
```

Comment:
You've set this as , so why are you insisting that it be explicitly set elsewhere?